### ISSUE TITLE
Debian package request

### ISSUE THREAD

**Issue by ura43** [qtile/qtile#3357]
It would be great to have a debian package for qtile again.
This would drastically ease the installation and lead to higher adoption in the long run.
It's important that the package is maintained and doesn't get abandoned like the [previous](https://tracker.debian.org/pkg/qtile) [two](https://launchpad.net/~tycho-s/+archive/ubuntu/ppa) attempts.

Some good entrypoints to relevant docs:
[debian new maintainers guide](https://www.debian.org/doc/manuals/maint-guide/)
[debian maintainers guide](https://www.debian.org/doc/manuals/debmake-doc/index.en.html)
[debian policy](https://www.debian.org/doc/debian-policy/)
[debian wiki packaging](https://wiki.debian.org/Packaging)

Existing packages:
[arch aur](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=qtile-git) (qtile 0.20.0)
[opensuse](https://build.opensuse.org/package/show/openSUSE%3AFactory/qtile) (qtile 0.18.1)
[freebsd](https://cgit.freebsd.org/ports/tree/x11-wm/qtile) (qtile 0.18.1)
[fedora](https://src.fedoraproject.org/rpms/qtile) (qtile 0.14.2)

_A maintained rpm package also wouldn't hurt_

**Comment by elParaguayo** [comment]
You need to pick this up with the debian team. We're not responsible for packaging for the distros here.

**Comment by ura43** [comment]
Yeah, thats not something you or mcol should ~~waste~~ spend your time on.

I can do some groundwork on this, but want to invite anyone looking for a way to contribute to leave a comment and contact me

> You need to pick this up with the debian team. We're not responsible for packaging for the distros here.
> 
> But I do agree, I'd like more distros to have this in their repos!

**Comment by ura43** [comment]
Can you give this the help wanted label?

**Comment by elParaguayo** [comment]
I might move this to a discussion as it's not a qtile "issue".
